<?xml version='1.0' ?>
<launch>
  <arg name='frequency' default='30.0' />
  <arg name='takeoff_height' default='10.0' />
  <arg name='odometry_topic' default='mavros/local_position/odom' />

  <node name='offboard_control' pkg='offboard_control' type='offboard_control' output='screen'>
    <param name='frequency' value='$(arg frequency)' />
    <param name='takeoff_height' value='$(arg takeoff_height)' />
    <param name='odometry_topic' value='$(arg odometry_topic)' />
  </node>
</launch>
